runtime/manta/src/lib.rs

```diff
@@ -104,7 +103,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("manta"),
 	impl_name: create_runtime_str!("manta"),
 	authoring_version: 1,
-	spec_version: 1,
+	spec_version: 3090,
```

Comment:
Shouldn't we start with `3000` in the manta runtime, since it's the shell chain runtime now.

Reply:
This is to be consistent with the release version. We are doing a similar semantic versioning on different runtimes with polkadot.